packages/material-ui/src/SnackbarContent/SnackbarContent.d.ts

```diff
@@ -3,7 +3,7 @@ import { StandardProps } from '..';
 import { PaperProps } from '../Paper';
 
 export interface SnackbarContentProps extends StandardProps<PaperProps, SnackbarContentClassKey> {
-  action?: React.ReactElement<any>;
+  action?: React.ReactElement<any> | React.ReactElement<any>[];
```

Comment:
Have you tried `React.ReactNode`?

Reply:
Thanks for the quick feedback!

I tried that just now, replacing the entire union with `React.ReactNode`, and it does resolve the issue. If that's the more appropriate/idiomatic fix, would it make sense to apply the same change to the `Snackbar` interface?

Reply:
Yes, and we can update `message` too.